.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on: [push]
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+
+        steps:
```

Comment:
Now we need to organize the build in the new way introduced by Ballerina.
To pack `pack ./simpledb` etc.

Reply:
Changed to latest way of packing bala files